docs/install_occlum_packages.md

```diff
@@ -0,0 +1,99 @@
+# Install Occlum Packages with Package Mangement Tool
+
+Occlum packages can be easily installed and deployed using mainstream package management tool including RPM and APT.
+
+
+### Prerequisite
+
+**Install enable_RDFSBASE Kernel Module**
+
+Please follow [this README](https://github.com/occlum/enable_rdfsbase/blob/master/README.md) to install `enable_rdfsbase` kernel module.
+
+
+### Install Occlum with RPM
+
+1. Install Prerequisite
+```
+yum install -y wget yum-utils make jq gdb
+```
+
+2. Install Intel® SGX Driver and Intel® SGX PSW
+
+Please follow [Intel SGX Installation Guide](https://download.01.org/intel-sgx/sgx-linux/2.11/docs/Intel_SGX_Installation_Guide_Linux_2.11_Open_Source.pdf) to install SGX driver and SGX PSW. SGX SDK is not required. Using RPM installer is recommanded. 
+
+Also, UAE service libraries are needed but may not installed together with SGX PSW if SGX PSW installer is used. Go to SGX RPM local repo and run:
+```
+rpm -i libsgx-uae-service*.rpm
+```
+
+After installing SGX driver and PSW, please make sure `aesm` service is running by checking:
+```
+service aesmd status
+```
+
+3. Install Occlum
+```
+cat << EOF > /etc/yum.repos.d/occlum.repo
+[occlum]
+name=occlum
+enabled=1
+baseurl=https://occlum.io/occlum-package-repos/rpm-repo/
+gpgcheck=1
+repo_gpgcheck=1
+gpgkey=https://occlum.io/occlum-package-repos/rpm-repo/RPM-GPG-KEY-rpm-sign
+gpgcakey=https://occlum.io/occlum-package-repos/rpm-repo/RPM-GPG-KEY-rpm-sign-ca
+EOF
+yum --showduplicate list -y occlum
+yum install -y occlum
+echo "source /etc/profile" >> $HOME/.bashrc
+```
+
+
+### Install Occlum with APT
+
+1. Install Prerequisite
+```
+apt-get update
+DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends ca-certificates gnupg2 jq make gdb wget libfuse-dev libtool
+```
+
+2. Install Intel® SGX Driver and Intel® SGX PSW
+
+Please follow [Intel SGX Installation Guide](https://download.01.org/intel-sgx/sgx-linux/2.11/docs/Intel_SGX_Installation_Guide_Linux_2.11_Open_Source.pdf) to install SGX driver and SGX PSW. SGX SDK is not required. Using PSW installer is recommanded.
+
+To install PSW, follow the guide to add Intel® SGX repository to APT source. And then run:
+```
+apt-get update
+apt-get install -y libsgx-epid libsgx-urts libsgx-quote-ex libsgx-uae-service
+```
+
+After installing PSW, please make sure that the aesm service is running:
+```
+service aesmd status
+```
+
+3. Install Occlum
+```
+echo 'deb [arch=amd64] https://occlum.io/occlum-package-repos/debian bionic main' | tee /etc/apt/sources.list.d/occlum.list
+wget -qO - https://occlum.io/occlum-package-repos/debian/public.key | apt-key add -
+apt-get update
+apt-cache policy occlum
+apt-get install -y occlum
+echo "source /etc/profile" >> $HOME/.bashrc
```

Comment:
I don't quite understand this line.

First, `/etc/profile` is the system-wide environment file that is used by all users and all shells (e.g., bash) on a system. On the other hand, `~/.bashrc` is the per-user config for bash. This means whatever inside the `/etc/profile` should have been applied to the current user's bash shell; no need to add it to `~/.bashrc` by hand.

Second, I am not aware of any Occlum's modifications to `/etc/profile`. So I don't know why we need to `source /etc/profile`.

Reply:
Occlum installers need to export some environment variables and those are done by a script in /etc/profile.d.
It's true that when the system start, the script in /etc/profile.d will all be executed. But some servers may never be shut down and the user newly login can get occlum command without reboot the system. Thus, put that to .bashrc seems a safer way.